bundles/org.openhab.binding.avmfritz/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -224,6 +224,22 @@
 		<state pattern="%s" readOnly="true"/>
 	</channel-type>
 
+	<channel-type id="obstruction_alarm">
+		<item-type>Contact</item-type>
+		<label>Obstruction Alarm</label>
+		<description>Obstruction alarm of the blinds. The blinds were stopped and moved a bit in the opposite direction.</description>
+		<category>Blinds</category>
+		<state pattern="%s" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="temperature_alarm">
```

Comment:
Is it normal to not find any usage of these channel types in a XML thing type ?

Reply:
No, I have added them now.